index.html

```diff
@@ -187,6 +187,41 @@ <h2>A Special time</h2>
             <a href="" class="social-net"><i class="fas fa-share"></i></a>
           </div>
         </div>
+        <div class="comments">
+          <form>
+            <fieldset>
+              <h2>What is your overall opinion about this article?</h2>
+              <div class="flex-container overal">
+                <label for="positive"><input type="radio" id="positive" value="positive">Positive </label>
+              <label for="negative"><input type="radio" id="negative" value="negative">Negative </label>
+              <label for="neutral"><input type="radio" id="neutral" value="neutral">Neutral </label>
+              </div>
+            </fieldset>
+            <fieldset>
+              <label class="xxl-font"><input type="checkbox"> I want to subscribe to the newsletter </label>
+            </fieldset>
+            <fieldset>
+              <div class="flex-container login">
+                <h2>Log In to add a comment</h2>
+                <a href="#">Not registered</a>
+              </div>
+              <div class="flex-container login">
+                <div class="wrapper">
+                  <input class="text" maxlength="20" type="text" placeholder="Email or Username" required><br>
+                  <input class="text" maxlength="20" type="text" placeholder="Password" required><br>
+                </div>
+                <div class="wrapper">
+                  <input class="btn" id="login" type="submit" value="Log In"><br>
+                  <input class="btn" id="sub" type="submit" value="Subscribe">
+                </div>
+              </div>
+            </fieldset>
+            <fieldset>
+              <textarea maxlength="250" class="flex-container" name="comment" id="comment" cols="30" rows="10" placeholder="Add a public comment..." disabled></textarea>
```

Comment:
Please use tag input with a type attribute of "textarea" instead of using <textarea> tag. This is required in the exercise.
Also, textarea input should be labeled with the text "Comment" or similar wording.

Reply:
I am sorry but there is no input tag with "textarea" type. That must be some mistake.